### ISSUE TITLE
Only standard Microsoft CRM Security Roles are download :(

### ISSUE THREAD

**Issue by Zeek2** [MscrmTools/MsCrmTools.SecurityRelated#8]
I like the look and operation of the  XrmToolBox's Privileges Discovery plugin, it seems fairly intuitive but was disappointed to find, when I clicked the "Display roles that match the selection" it only listed the 3 standard Microsoft Roles: Host System Administrator, System Administrator and System Customizer.  We have many more sercurity roles that we have defined and they are more important for us - and there is no sign of them in the plugin.

Please download all Security Roles, including non-standard/custom security roles.

**Comment by MscrmTools** [comment]
The plugin does display custom roles (see example below)

![image](https://user-images.githubusercontent.com/10774317/43457606-af266420-94c7-11e8-96ad-f36b703b0f95.png)

**Comment by Zeek2** [comment]
Yes, something like that is what I expected to see (albeit with a much longer list) but we just get the 3 standard security roles:
[](url)

**Comment by MscrmTools** [comment]
Are you system administrator on your CRM organization? Only root roles are retrieved (the ones in the top BU)

**Comment by Zeek2** [comment]
Yes, I was connected as the System Admin.   I just connected as a Tenant Admin with the same result.

**Comment by MscrmTools** [comment]
Can you show a screen capture of one of your attempt to load more roles than what is actually displayed?
